dnf/cli/commands/install.py

```diff
@@ -127,3 +129,17 @@ def run(self):
             raise dnf.exceptions.PackagesNotAvailableError(
                 _("Unable to find a match"), pkg_spec=' '.join(errs),
                 packages=err_pkgs)
+
+    def check_arguments(self):
+        security_filters = [self.opts.bugfix, self.opts.enhancement, self.opts.newpackage,
+                            self.opts.security, self.opts.advisory, self.opts.bugzilla,
+                            self.opts.cves, self.opts.severity]
+        specs = [self.opts.pkg_specs, self.opts.grp_specs, self.opts.filenames]
+        if any(security_filters):
+            if all(not spec for spec in specs):
+                self.opts.pkg_specs.append("*")
+            if len(self.base._update_security_filters["minimal"][0]) == 0:
+                raise dnf.exceptions.Error(_("No security updates to install"))
+        elif all(not spec for spec in specs):
+            raise dnf.exceptions.Error(_("dnf install: the following arguments "
```

Comment:
No usage of command will be printed to terminal. Somehow I am not fan of optional argument here (I prefer "+" insted of "*"), because it is mass-destruction weapon. Just think about command "dnf install --security -y". Possibility without an argument I would like to implement only on user request.